terraform/aks/workspace_variables/dv_review.tfvars.json

```diff
@@ -0,0 +1,11 @@
+{
+  "environment_name": "review",
+  "cluster": "cluster2",
+  "namespace": "development",
+  "resource_group_name": "s189d01-trs-rv-rg",
+  "enable_monitoring": false,
+  "deploy_dqt_reporting_server": true,
+  "run_dqt_reporting_service": true,
+  "run_recurring_jobs": false,
+  "enable_logit": true
+}
```

Comment:
deploy_azure_backing_services should be false

Reply:
added this variable as false